CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,129 @@
+
+# Contributor Covenant Code of Conduct
```

Comment:
Is this actually just the Contributor Covenant coc or are there mods. At the end it says this has been adapted from the Covenant one?  I'd prefer this just to "Code of Conduct" or "TODO Group Code of Conduct". It's odd to go to group A and be presented with rules about how group B operates.

Reply:
To be specific, this is a copy of the markdown version of the [Contributor Covenant 2.0](https://www.contributor-covenant.org/version/2/0/code_of_conduct/code_of_conduct.md). There were two trivial modifications, which I will call out below. This title, as well as the Attribution section, are unmodified from the original document.

I'd argue that keeping the title above as "Contributor Covenant Code Of Conduct" aids in two ways.
* People looking for a code of conduct are likely to be familiar with this one, which will make it easy to identify
* It makes it clear that we didn't roll it ourselves.

Reply:
Good points. In that sense it can be seen as more like a templated license. We are taking the base template, updating it in the expected ways and using it. If we were doing more significant (or unexpected) changes then it would no longer be the same.